src/e2e-test/features/gcs/source/GCSSourceError.feature

```diff
@@ -74,14 +61,5 @@ Feature: GCS source - Verify GCS Source plugin error scenarios
     Then Select GCS property format "csv"
     Then Enter GCS source property regex path filter "gcsIncorrectRegexPath"
     Then Toggle GCS source property skip header to true
-    Then Validate output schema with expectedSchema "gcsCsvFileSchema"
-    Then Validate "GCS" plugin properties
-    Then Close the GCS properties
-    Then Open BigQuery sink properties
-    Then Override Service account details if set in environment variables
-    Then Enter the BigQuery sink mandatory properties
-    Then Validate "BigQuery" plugin properties
-    Then Close the BigQuery properties
-    Then Save the pipeline
```

Comment:
Similar question as above

Reply:
yeah, now the plugin fails in validation rather than going to the preview stage.